services/src/main/java/org/apache/druid/cli/CliHadoopIndexer.java

```diff
@@ -84,9 +86,13 @@ public void run()
         extensionURLs.addAll(Arrays.asList(extensionLoader.getURLs()));
       }
 
-      final List<URL> nonHadoopURLs = Arrays.asList(
-          ((URLClassLoader) CliHadoopIndexer.class.getClassLoader()).getURLs()
-      );
+      final List<URL> nonHadoopURLs;
+      final ClassLoader indexerLoader = CliHadoopIndexer.class.getClassLoader();
```

Comment:
We have removed java 8 support. So we donot need to support JDK 8 and below. so the if on line 91 can be removed.